PriorityQueue contstructor's default initialization wrong?

We are trying to migrate [Seg3D](http://www.seg3d.org/) to ITKv5. I am currently stuck at [MinPriorityQueueElementWrapper's constructor](https://github.com/dzenanz/Seg3D/blob/41c7871198d114360c0a394ca993a50f59898174/src/Core/ITKLiveWire/itkLiveWireImageFunction.hxx#L218) failing to compile. Any opinions about whether this a problem in Seg3D or ITK? In particular, should [m_Element(0)](https://github.com/InsightSoftwareConsortium/ITK/blob/v5.1.0/Modules/Core/Common/include/itkPriorityQueueContainer.hxx#L94) be `m_Element({})` (it compiles with this change) or something similar?

Complete error message:
```text
ActionSpeedline.cc
m:\dev\seg3d-vs17sb\externals\install\itk_external\include\itk-5.1\itkPriorityQueueContainer.hxx(94): error C2664: 'itk::Index<2>::Index(itk::Index<2> &&)': cannot convert argument 1 from 'int' to 'const itk::Index<2> &'
m:\dev\seg3d-vs17sb\externals\install\itk_external\include\itk-5.1\itkPriorityQueueContainer.hxx(97): note: Reason: cannot convert from 'int' to 'const itk::Index<2>'
m:\dev\seg3d-vs17sb\externals\install\itk_external\include\itk-5.1\itkPriorityQueueContainer.hxx(97): note: No constructor could take the source type, or constructor overload resolution was ambiguous
m:\dev\seg3d-vs17sb\externals\install\itk_external\include\itk-5.1\itkPriorityQueueContainer.hxx(94): note: while compiling class template member function 'itk::MinPriorityQueueElementWrapper<itk::Index<2>,itk::LiveWireImageFunction<TYPED_IMAGE_TYPE_2D>::RealType,itk::LiveWireImageFunction<TYPED_IMAGE_TYPE_2D>::ElementIdentifier>::MinPriorityQueueElementWrapper(void)'
m:\dev\seg3d-vs17sb\externals\install\itk_external\include\itk-5.1\itkPriorityQueueContainer.hxx(272): note: see reference to function template instantiation 'itk::MinPriorityQueueElementWrapper<itk::Index<2>,itk::LiveWireImageFunction<TYPED_IMAGE_TYPE_2D>::RealType,itk::LiveWireImageFunction<TYPED_IMAGE_TYPE_2D>::ElementIdentifier>::MinPriorityQueueElementWrapper(void)' being compiled
m:\dev\seg3d-vs17sb\externals\install\itk_external\include\itk-5.1\itkPriorityQueueContainer.hxx(105): note: while compiling class template member function 'itk::MinPriorityQueueElementWrapper<itk::Index<2>,itk::LiveWireImageFunction<TYPED_IMAGE_TYPE_2D>::RealType,itk::LiveWireImageFunction<TYPED_IMAGE_TYPE_2D>::ElementIdentifier>::MinPriorityQueueElementWrapper(itk::Index<2>,float)'
m:\dev\seg3d\src\core\itklivewire\itkLiveWireImageFunction.hxx(218): note: see reference to function template instantiation 'itk::MinPriorityQueueElementWrapper<itk::Index<2>,itk::LiveWireImageFunction<TYPED_IMAGE_TYPE_2D>::RealType,itk::LiveWireImageFunction<TYPED_IMAGE_TYPE_2D>::ElementIdentifier>::MinPriorityQueueElementWrapper(itk::Index<2>,float)' being compiled
m:\dev\seg3d\src\core\itklivewire\itkLiveWireImageFunction.hxx(218): note: see reference to class template instantiation 'itk::MinPriorityQueueElementWrapper<itk::Index<2>,itk::LiveWireImageFunction<TYPED_IMAGE_TYPE_2D>::RealType,itk::LiveWireImageFunction<TYPED_IMAGE_TYPE_2D>::ElementIdentifier>' being compiled
m:\dev\seg3d\src\core\itklivewire\itkLiveWireImageFunction.hxx(130): note: while compiling class template member function 'void itk::LiveWireImageFunction<TYPED_IMAGE_TYPE_2D>::GeneratePathDirectionImage(void)'
m:\dev\seg3d\src\core\itklivewire\itkLiveWireImageFunction.h(168): note: see reference to function template instantiation 'void itk::LiveWireImageFunction<TYPED_IMAGE_TYPE_2D>::GeneratePathDirectionImage(void)' being compiled
M:\Dev\Seg3D\src\Application\Tools\Actions\ActionSpeedline.cc(195): note: see reference to class template instantiation 'itk::LiveWireImageFunction<TYPED_IMAGE_TYPE_2D>' being compiled
M:\Dev\Seg3D\src\Application\Tools\Actions\ActionSpeedline.cc(171): note: see reference to function template instantiation 'void Seg3D::ActionSpeedlineAlgo::typed_run_filter<signed char>(void)' being compiled
M:\Dev\Seg3D-vs17sb\Externals\Source\Boost_external\boost/bind/placeholders.hpp(54): note: see reference to class template instantiation 'boost::arg<9>' being compiled
M:\Dev\Seg3D-vs17sb\Externals\Source\Boost_external\boost/bind/placeholders.hpp(53): note: see reference to class template instantiation 'boost::arg<8>' being compiled
M:\Dev\Seg3D-vs17sb\Externals\Source\Boost_external\boost/bind/placeholders.hpp(52): note: see reference to class template instantiation 'boost::arg<7>' being compiled
M:\Dev\Seg3D-vs17sb\Externals\Source\Boost_external\boost/bind/placeholders.hpp(51): note: see reference to class template instantiation 'boost::arg<6>' being compiled
M:\Dev\Seg3D-vs17sb\Externals\Source\Boost_external\boost/bind/placeholders.hpp(50): note: see reference to class template instantiation 'boost::arg<5>' being compiled
M:\Dev\Seg3D-vs17sb\Externals\Source\Boost_external\boost/bind/placeholders.hpp(49): note: see reference to class template instantiation 'boost::arg<4>' being compiled
M:\Dev\Seg3D-vs17sb\Externals\Source\Boost_external\boost/bind/placeholders.hpp(48): note: see reference to class template instantiation 'boost::arg<3>' being compiled
M:\Dev\Seg3D-vs17sb\Externals\Source\Boost_external\boost/bind/placeholders.hpp(47): note: see reference to class template instantiation 'boost::arg<2>' being compiled
M:\Dev\Seg3D-vs17sb\Externals\Source\Boost_external\boost/bind/placeholders.hpp(46): note: see reference to class template instantiation 'boost::arg<1>' being compiled
Done building project "Application_Tools.vcxproj" -- FAILED.
```

--- a/Modules/Core/Common/include/itkPriorityQueueContainer.hxx
+++ b/Modules/Core/Common/include/itkPriorityQueueContainer.hxx
@@ -91,9 +91,7 @@ const TElementIdentifier ElementWrapperPointerInterface<TElement, TElementIdenti
 // -----------------------------------------------------------------------------
 template <typename TElement, typename TElementPriority, typename TElementIdentifier>
 MinPriorityQueueElementWrapper<TElement, TElementPriority, TElementIdentifier>::MinPriorityQueueElementWrapper()
-  : m_Element(NumericTraits<TElement>::ZeroValue())
-  , m_Priority(NumericTraits<TElementPriority>::ZeroValue())
-  , m_Location(Superclass::m_ElementNotFound)
+  : m_Location(Superclass::m_ElementNotFound)
 {}
 // -----------------------------------------------------------------------------
 

